### DISCUSSION TITLE
Open source

### DISCUSSION THREAD

**Discussion by Camille-SC** [thegetty/quire#202] [General]
Hi
I read with interest, in the Erin interview, the project about  launching Quire as a fully open source tool. What about Prince XML ? Will it be replaced?
Thank you

**Comment by Erin-Cecele** [comment]
Hi @Camille-SC, Glad you enjoyed the MuseumNext interview. 

When we say Quire will be launched as "fully open source" we mean that it will have an official OS license. Prince will remain the one proprietary software that is used in the Quire process, but we definitely want to pursue open source options for PDF generation in the future. We just don't have a timeline for an actual replacement at this point. 

One option we have been considering is **paged.js** which is open source. I believe, last we discussed, you were considering testing it out. Have you had any luck?

**Reply by Camille-SC** [reply]
Hi Erin,
Thank you very much for your reply. Yes, we are considering it (this kind of new project is so desperatly long in here, and the covid stopped a lot of things this past monthes), and we are now in touch with one of the creators of paged.js. The question of the license and the governance of the tools is a main point! Anyway, I truly hope that our effort will achieved to sucess and be the largest useful  as possible for the community of the museums and publishers.
I always read the Quire newsletter with pleasure and interest ;-)
Best, Camille

**Reply by Erin-Cecele** [reply]
Thank you, @Camille-SC. Please do keep us updated. And, we're so glad to hear you enjoy the newsletter :-). We look forward to showcasing your publication in it one day.
